src/pages/ProductList/component/FilterDetail.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import './FilterDetail.scss';
+
+const FilterDetail = ({ name, check }) => {
+  const [isCheck, setIsCheck] = useState(false);
+  return (
+    <div className="filter-detail">
+      <div className="filter-header">
+        <p className="filter-name">{name}</p>
+        {isCheck ? (
+          <img
+            className="sort-img"
+            alt="더보기"
+            src="images/icon/angle-up-solid.svg"
+            onClick={() => setIsCheck(prev => !prev)}
+          />
+        ) : (
+          <img
+            className="sort-img"
+            alt="더보기"
+            src="images/icon/angle-down-solid.svg"
+            onClick={() => setIsCheck(prev => !prev)}
+          />
+        )}
+      </div>
+      {isCheck ? (
+        check ? (
```

Comment:
```suggestion
        check && (
```
이렇게하고, 저 아래에 null은 없어도 되겠네요
그리고 props 이름이 check는 조금 모호해서, category나 option 은 어떨까요?